health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MetricChart.js

```
@@ -0,0 +1,154 @@
+  const getHeight = (data) => {
+    if (isMobile) return "auto";
+    else return 50;
+    // if (isMobile) return t(`TIP_${data.name}`).length < 50 ? 50 : "auto";
+    // else return 50;
+  }
+
+  return (
+    <div className="row">
+      <div className={`tooltip`} >
+        {typeof name=="string"&&name}
+        {Array.isArray(name)&&name?.filter(ele=>ele)?.map(ele=><div style={{ whiteSpace: "pre" }}>{ele}</div>)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)